TrainingJava/src/ShoppingCartCounter/PaymentMethodFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package ShoppingCartCounter;
+
+/**
+ *
+ * @author andreslietti
+ */
+
+
+//@author andreslietti
+
+public class PaymentMethodFactory {
+
+    public PaymentMethod getMethod (String method){
+        PaymentMethod paymentMethod = null;
+
+        switch (method){
+            case "Credit Card":
+                paymentMethod = new CreditCard();
+                break;
+
+            case "PayPal":
+                paymentMethod = new PayPal();
+                break;
+
+            case "Cash":
+                paymentMethod = new Cash();
+                break;
+
+            default:
+                return null;
```

Comment:
We are already returning null, if anything matches in the switch statement.
just add brea; and delete the return null;